foundation/okfplugins/heading/templates/heading_plugin.html

```diff
@@ -1,6 +1,14 @@
 {% load markdown_deux_tags %}
 <!-- heading -->
 <div class="headline -tighten-1 text-center mb-20">
-  <h1 class="col-span-full px-3 py-5 font-{% if instance.highlighted %}mono{% else %}bold{% endif %} text-{{ instance.alignment }} text-{{ instance.heading_type }}">{{ instance.title | markdown }}</h1>
+  <h1 class="col-span-full px-3 py-5 font-{% if instance.highlighted %}mono{% else %}bold{% endif %} text-{{ instance.alignment }} text-{{ instance.heading_type }}">
+    {% if instance.is_anchor %}
+    <a id="{{ instance.anchor_id }}">
+    {% endif %}
+      {{ instance.title | markdown }}
+    {% if instance.is_anchor %}
+    </a>
+    {% endif %}
```

Comment:
I think in this case is better, for readability, to use a syntaxis like this:
```
{% if instance.anchor_id %}
  <a id="{{ instance.anchor_id }}">{{ instance.title | markdown }}</a>
{% else %}
  {{ instance.title | markdown }}
{% endif %}
```

Even when we are repeating `{{ instance.title | markdown }}` it is betther to not spllit the `<a>` element.

Reply:
yes! this is better.